### PR TITLE
adding API queries

### PR DESCRIPTION
**Pull request by avkr003**

### CODE REVIEW

File: x/masterchef/keeper/query_chain_tvl.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	stakedEden := commitmentParams.TotalCommitted.AmountOf(ptypes.Eden)
+	stakedEdenValue := elysPrice.MulInt(stakedEden)
+	totalTVL = totalTVL.Add(stakedEdenValue.TruncateInt())
+
+	return &types.QueryChainTVLResponse{
+		Total:       totalTVL,
+		Pools:       poolsTVL.TruncateInt(),
+		UsdcStaking: stableStakeTVL.TruncateInt(),
+		StakedElys:  stakedElysValue.TruncateInt(),
+		StakedEden:  stakedEdenValue.TruncateInt(),
```
Comment (amityadav0) [inline x/masterchef/keeper/query_chain_tvl.go:60]: We can add stakedEdenB as well

File: x/masterchef/keeper/query_chain_tvl.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	}
+	totalTVL = totalTVL.Add(poolsTVL.TruncateInt())
+
+	baseCurrencyEntry, found := k.assetProfileKeeper.GetEntry(ctx, ptypes.BaseCurrency)
+	if !found {
+		return nil, status.Error(codes.NotFound, "asset profile not found")
+	}
+
+	stableStakeTVL := k.stableKeeper.TVL(ctx, k.oracleKeeper, baseCurrencyEntry.Denom)
+	totalTVL = totalTVL.Add(stableStakeTVL.TruncateInt())
```
Comment (amityadav0) [inline x/masterchef/keeper/query_chain_tvl.go:42]: this seems to be repeated code, can you reuse the function above for this ?

Comment (avkr003) [reply x/masterchef/keeper/query_chain_tvl.go:42]: two different queries, one is chain tvl and other pool tvl query as required by defi lama

File: x/masterchef/keeper/query_chain_tvl.go (hunk 1 above)
Comment (avkr003) [reply x/masterchef/keeper/query_chain_tvl.go:60]: EdenB is not required and doesn't mint to Elys after vesting as required by defi lama

**Review by amityadav0 — APPROVED** [review]

**Pull request merged**